Jenkinsfile

```diff
@@ -4,8 +4,42 @@ properties(
   ]
 )
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
 node('buildvm-devops') {
-  sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
-    echo "Bruno's branch split pipeline"
+  try {
+    deleteDir()
+    set_workspace()
+    stage('clone') {
+      dir('aos-cd-jobs') {
+        checkout scm
```

Comment:
That is super weird. On the freestyle jobs if you do a SCM trigger on a commit, it does the cloning for you before your steps run. They really turned everything on it's head with the Jenkinsfiles :\